python/pyspark/sql/dataframe.py

```diff
@@ -1508,7 +1518,7 @@ def toDF(self, *cols):
         return DataFrame(jdf, self.sql_ctx)
 
     @since(1.3)
-    def toPandas(self):
+    def toPandas(self, useArrow=True):
```

Comment:
Would it maybe make more sense to default this to false or have more thorough checking that the dataframe being written with arrow is supported? At least initially the set of supported dataframes might be rather small.